entries/entry04.md

```diff
@@ -1,8 +1,23 @@
 # Entry 4
```

Comment:
Sources: 0
EDP: 1
Knowledge: 1
Skills: 0
Professionalism: 0

In the next entry, I want to see how your models are being included in the overall game. That's part of working towards the MVP! These models are useless if you can't get them in the actual game.

Also, use the rubric!